src/backend/routers/user.py

```diff
@@ -88,10 +90,15 @@ async def login(
     password = user_login.password
 
     user: User | None = await User.prisma().find_first(where={"username": username})
-
-    if user is None or (not await check_password(password, user.password)):
+    if user is None:
+        raise HTTPException(status_code=404, detail="User does not exist")
```

Comment:
Can you change the error response to "The username or password is incorrect." for security:tm:?

Reply:
Resolved in [0f7fac6](https://github.com/idkbrowby/gamingrealm-backend/pull/31/commits/0f7fac6a4c34679f95618824a5856a2f5dc894c5)